templates/partials/sender.html

```diff
@@ -4,10 +4,10 @@
     ng-bind="message.contact.name || (!message.form && message.name) || message.from || message.sent_by"></span>
     <span class="phone"
     ng-hide="message.name && message.name === message.facility.contact.phone"
-    ng-bind="message.contact.phone"></span>
+    ng-bind="message.contact.phone || message.phone"></span>
```

Comment:
No this is relevant to this one. Specifically it's so we can render the report summary or full details for performance reasons. See #2302